proposals/custom-descriptors/Overview.md

```diff
@@ -648,7 +650,7 @@ describing the methods that should be attached to the prototype
 after instantiation.
 Each configured method can be either a
 normal method, a getter, a setter, or a constructor.
-Methods also have two associated names: the first their property name
+Methods each have two associated names: the first their property name
```

Comment:
While you're here, the second half of the line around the word "first" could also do with a grammar fix: either drop "the" to make it an enumeration, or insert "is" to make it a sentence. Same below, around "second".